builds/misc/packages-release.yaml

```diff
@@ -241,7 +245,11 @@ stages:
             artifactName: iotedged-centos7-amd64
             identityServiceArtifactName: packages_centos-7_amd64
             identityServicePackageFilter: aziot-identity-service-*.x86_64.rpm
-          
+          RedHat8-amd64:
+              os: redhat8
```

Comment:
We would need to update https://github.com/Azure/iotedge/blob/4a6ae13e283e006cd99a975d9a54bc7364b4c83e/scripts/linux/publishReleasePackages.sh#L51
to add support for RHEL8 :)